src/universal/containers/DashLayoutContainer/DashLayoutContainer.js

```diff
@@ -81,15 +83,28 @@ export default class DashLayoutContainer extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    const {activeMeetings, trialNotification} = this.props;
+    const {barType} = trialNotification;
     if (this.props.tms !== nextProps.tms) {
       subToAllTeams(nextProps.tms);
     }
+
+    if (activeMeetings.length > 0 || barType === TRIAL_EXPIRES_SOON || barType === TRIAL_EXPIRED) {
+      this.props.dispatch(notificationBarPresent(true));
+    } else {
+      this.props.dispatch(notificationBarPresent(false));
```

Comment:
can we grab `hasNotificationBar` and ONLY call dispatch if nextProps.hasNotificationBar !== this.props.hasNotificationBar? Otherwise we'll be triggering a dispatch on every new prop that comes in